### PR TITLE
Added a SteelSheet and GlassSheet reactions to fun.yml to go along with the PlasticSheet reaction

### PR DESCRIPTION
**Pull request by luizwritescode**
## About the PR

**Disclaimer**: This is my first PR and a learning experience to get used to the codebase and git colaboration. I do not expect this to get merged, however it would be _really_ cool if it did. 

Currently, you can make plastic as a chemical reaction. Why not include Steel and Glass as well? This adds two new reactions to the fun.yml file right below the Plastic reaction.

## Why / Balance

Plastic reaction is not really economic as we don't players to spam materials round start in Chemistry. It is not their job. Materials made in a beaker should be made as a last resort if Cargo is not operational or if you can't find things to salvage.

The plastic reaction reagents add up to 10 total to create a single sheet so I followed that formula to balance Glass and Steel. I tried loosely basing them on real life chemical reactions. It also gives more gameplay value to the Iron chemical.

Recipes:

1 Steel sheet:
 - 9 iron
 - 1 carbon

1 Glass sheet:
 - 3 silicon
 - 6 oxygen
 - 1 water

## Sidenote

if this gets implemented maybe move them to their own materials.yml file because they don't really fit theme of "fun"

**Changelog**

🆑 
- add: Steel Sheets and Glass sheets can now be created by heating chemicals in a beaker

### CODE REVIEW

**Comment by luizwritescode** [comment]
someone told me the recipes are too basic and now that I see it, I totally agree... accepting suggestions because only thing I can think of is increasing the reaction temperature so it takes longer (like approximately the time it would take to make SulfuricAcid, Oil and burn a paper for Ash)

**Comment by bVelliquette** [comment]
[Maybe this could be good inspiration on how to complicate the glass recipe a bit.](https://en.wikipedia.org/wiki/Chemically_strengthened_glass) I think it makes sense that the glass used in construction of a space station would need to be chemically strengthened.

File: Resources/Prototypes/Recipes/Reactions/fun.yml [hunk 1]
```diff
@@ -93,7 +93,7 @@
     Licoxide: 1
 
 - type: reaction
-  id: PlasticSheet # It's here because "haha look you can get smooth beautiful sheets of plastic from small cylindric beaker" (jokes aside: i dont know where should i put it)
+  id: PlasticSheet # plastic steel and glass should have their own .yml file named "materials.yml". also note how all material reactions total to 10 reactants to follow balance
```
Comment (beck-thompson) [inline Resources/Prototypes/Recipes/Reactions/fun.yml:96]: Why don't you do that in this PR? Unless I'm missing something (Easily possible!) you should just be able to create that file in this directory and add the three reactions to it!

Comment (luizwritescode) [reply Resources/Prototypes/Recipes/Reactions/fun.yml:96]: I could do that as a last commit for organization. The problem is this whole PR is currently dividing the contributors on whether chemistry should be an alternative (but expensive) source of materials should cargo not be available. One one side Botany can currently make Steel, on the other side it is detrimental to both salvage and cargo balance as it undermines their purpose.

**Comment by luizwritescode** [comment]
Rebalanced the reactions and added an intermediate reactant for Glass, named "Sand". This should at least make the reactions not as commonly made in-game and as a true "last restort" for materials should cargo not be available.

New recipes:

4 Sand:
 - 1 Silicon
 - 2 Oxygen
 - 1 TableSalt

1 GlassSheet:
 - 5 Sand
 - 3 Oxygen
 - 2 Potassium

1 SteelSheet:
 - 5 Iron
 - 3 Fersilicite
 - 2 SodiumCarbonate


However, @Blackern5000 caught my oversight as we can't have a Silicon as an ingredient if GlassSheet will grind to more than it is required to create as that is most definitely a dupe glitch.

**Comment by thebadman4662** [comment]
Frankly I can't think of situation where anyone ever runs out of silicon, so as long as it is not main ingredient there should be no problem with it.

Oxygen, carbon, water, sugar, potassium are only things that you can expect chemlab to run out of and a lot of it can easily be harvested too.

**Comment by thebadman4662** [comment]
> > Oxygen, carbon, water, sugar, potassium are only things that you can expect chemlab to run out of and a lot of it can easily be harvested too.
> 
> oxygen and water are infinite, you can get water from sinks and electrolyze it into hydrogen and oxygen.

Time is not infinite though, you can very easily run out of oxygen and water fast, especially if condenser is not mapped in, atmos refuse to cooperate and you can't get a board from science for whatever reason. Of course its rare that all 3 happen, but every now and then all 3 do when its heavy use of cryox shift from hell from the very start.

**Review by UbaserB — CHANGES_REQUESTED** [review]
Could you make the reagents required to create the steel and glass be the same as what you obtain when you grind steel and glass? It should be 1:1 as per the law of conservation of mass. Otherwise, feel free to keep the recipe as difficult as it currently is.

**Comment by UbaserB** [comment]
You coming back to this?

**Comment by UbaserB** [comment]
Feel free to reopen if you want to come back to this.

**Pull request closed without merging**